### PR TITLE
Fixes logs when making a tyypo in a command

### PR DESCRIPTION
**Pull request by abeisgoat**
The winston transports never got initialized when command routing fell back to the default route so I generalized the code and manually initialized it when we're not using the `Command` class (where it's normally `add`'d)

### CODE REVIEW

File: src/utils.ts [hunk 1]
```diff
@@ -5,6 +5,9 @@ import { Readable } from "stream";
 import { configstore } from "./configstore";
 import { FirebaseError } from "./error";
 import * as logger from "./logger";
+import * as winston from "winston";
+import { SPLAT } from "triple-beam";
+const ansiStrip = require("cli-color/strip") as (input: string) => string;
```
Comment (bkendall) [inline src/utils.ts]: Since these are all external modules, I'd group them above with the other external modules.

Comment (abeisgoat) [reply src/utils.ts]: good catch, auto import always doing me dirty

**Comment by samtstern** [comment]
@abeisgoat did you put a typo in this PR title on purpose?

**Comment by abeisgoat** [comment]
yes

**Pull request merged**